dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/events/EventNotesTests.java

```diff
@@ -76,7 +76,7 @@ public void shouldUpdateEventWithANote()
         trackerActions.getEvent( eventId + "?fields=notes" )
             .validate().statusCode( 200 )
             .body( "notes", hasSize( 2 ) )
-            .body( "notes.storedBy", CoreMatchers.everyItem( equalTo( "taadmin" ) ) );
```

Comment:
Was storedBy on notes the "same type" of storedBy that is used to store all kinds of things on other entities like trackers  Event.storedBy?

Reply:
👍